packages/client/src/pages/sign-up.tsx

```diff
@@ -1,31 +1,78 @@
 import React from "react";
-
+import * as Yup from "yup";
+import { useRouter } from "next/router";
+import { useCreateUserMutation } from "../generated/codegen";
 import {
   Formik,
   Center,
   Button,
   Form,
   InputField,
   Heading,
+  useToast,
 } from "@mustardmind/mauinz";
+import { toErrorMap } from "src/util/toErrorMap";
 
-interface SignUpFormValues {
+interface SignUpFormFields {
   email: string;
   password: string;
+  name: string;
+  phone: string;
 }
```

Comment:
we don't need this interface, there's a generated one by the code-generator, I guess its called `UserInput`.